docs/pages/heroes.md

```diff
@@ -230,6 +230,139 @@ variation_groups:
+                          50 chars max for a two-line heading
+                      </p>
+                      <p class="m-hero_subhead">    
+                          This text has a recommended count of 140-175 characters (four lines at 1230px) following a two- or three-line heading.
+                      </p>
+                  </div>
+                  <div class="m-hero_image-wrapper">
+                      <div class="m-hero_image"></div>
+                  </div>
+              </div>
+              <style>
+                  #example-jumbo-hero .m-hero_image {
+                      background-image:
+                          url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg');
+                      background-image: -webkit-image-set(
+                          url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 1x,
+                          url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 2x
+                      );
+                      background-image: image-set(
+                          url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 1x,
+                          url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 2x
+                      );
+                      filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
+                          src='/design-system/images/uploads/jumbo_hero_image_wide.jpeg',
+                          sizingMethod='scale');
+                      padding-bottom: 41.6666667%;
+                  }
+
+                  @media screen and (min-width: 37.5625em) {
+                      #example-jumbo-hero .m-hero_wrapper {
+                          background-image:
+                              url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg');
+                          background-image: -webkit-image-set(
+                              url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 1x,
+                              url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 2x
+                          );
+                          background-image: image-set(
+                              url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 1x,
+                              url('/design-system/images/uploads/jumbo_hero_image_wide.jpeg') 2x
+                          );
+                          filter: progid:DXImageTransform.Microsoft.AlphaImageLoader(
+                              src='/design-system/images/uploads/jumbo_hero_image_wide.jpeg',
+                              sizingMethod='scale');
+                      }
+                  }
+              </style>
+          </section>
+        variation_code_snippet_rendered: >-
+          <section class="m-hero m-hero__jumbo" id="example-jumbo-hero">
+              <div class="m-hero_wrapper wrapper">
+                  <div class="m-hero_text">
+                      <p class="h1 m-hero_heading">
```

Comment:
@contolini sorry I missed this discussion -- these were `<p>` to avoid a Lighthouse issue with out-of-order headings on the page. See https://github.com/cfpb/design-system/pull/1328.